host/xtest/pkcs11_1000.c

```diff
@@ -146,6 +151,92 @@ static void xtest_tee_test_1001(ADBG_Case_t *c)
 	}
 
 	Do_ADBG_EndSubCase(c, "Test C_Get{Slot|Token}Info()");
+	Do_ADBG_BeginSubCase(c, "Test C_GetMechanism{List|Info}()");
+
+	for (i = 0; i < slot_count; i++) {
+		CK_SLOT_ID slot = slot_ids[i];
```

Comment:
What's in this loop should probably go into a helper function instead.

Reply:
Ok I you think it helps.